### PR TITLE
Fix Missing "Time left" in DB when queue is cleared

### LINKED ISSUE #49: DB Missing Time Left
## Description

Some entries in the visits table are missing their "time_left" field. This was a problem in V2 as well and was carried over.
I found that when the queue is cleared the visit entries aren't updated. That should be fixed.
Also need to check the other ways in which a student could be removed from the queue but not have the corresponding visit
entry updated.

## Steps to Reproduce

Look through stats file after clearing the queue.

## Suspected Cause

My bad code lol
Should refactor the logic for removing a student into one place.

### PR DESCRIPTION
**Pull request by SamLab17**
Refactors a bit of the logic for removing students from the queue and when the queue is cleared, the students removed have the correct timestamp added to "time left" in the database.

Fixes #49

### CODE REVIEW

File: helphours/routes_helper.py [hunk 1]
```diff
@@ -24,34 +22,21 @@ def handle_line_form(request, curr_open_state):
...
-            v.was_helped = 1
-            v.instructor_id = g.user.id
-        elif 'removed' in request.form:
-            v.was_helped = 0
-        # Write changes to database
-        db.session.commit()
-    else:
-        print(f"Did not find entry for {uid} in the visits table.")
+    # Remove the student from the queue and update the database
+    remove_helper(uid, was_helped=('finised' in request.form), instructor_id=g.user.id)
```
Comment (ClosingBracket) [inline helphours/routes_helper.py]: finished

File: helphours/routes_helper.py [hunk 2]
```diff
@@ -83,3 +66,22 @@ def get_place_str(place):
...
+    if v is not None:
+        v.time_left = datetime.now()
+        if was_helped:
+            v.wasHelped = 1
+            v.instructor_id = instructor_id
+        else:
+            v.wasHelped = 0
+        db.session.commit()
+    else:
+        log.error(f"DId not find entry for {uid} in the visits table.")
```
Comment (ClosingBracket) [inline helphours/routes_helper.py]: typo

**Review by ClosingBracket — CHANGES_REQUESTED** [review]

File: helphours/routes_helper.py (hunk 2 above)
Comment (SamLab17) [reply helphours/routes_helper.py]: bro get outta here with your outdated requests

**Review by ClosingBracket — APPROVED** [review]
Nice!

**Review by ninaahmed — APPROVED** [review]
Nice

**Pull request merged**